src/main/groovy/com/palantir/gradle/javadist/tasks/DistTarTask.groovy

```diff
@@ -40,7 +40,18 @@ class DistTarTask extends Tar {
                 }
             }
 
-            new File(project.projectDir, "var/data/tmp").mkdirs()
+            // Create var/data/tmp
+            File.createTempDir().with { emptyTempDirBase ->
+                emptyTempDirBase.deleteOnExit()
+                emptyTempDirBase.toPath().resolve("data").resolve("tmp").toFile().mkdirs()
+                from(emptyTempDirBase) {
+                    into "${archiveRootDir}/var"
+                    // TODO(rfink) If users exclude 'data', then the temp dir will not be created. That seems wrong?
```

Comment:
Per https://github.com/palantir/gradle-java-distribution/issues/114#issuecomment-246156725 I think we want to just have it be `var/tmp`, and then you don't have to worry about this.